tools/convert_background.py

```diff
@@ -26,7 +26,7 @@ def write_result(bytes, outfile, wrap_count=None):
     operation_args = arg_parser.add_argument_group('Operation').add_mutually_exclusive_group()
     operation_args.add_argument('--decode', '-d', nargs=1, type=str, metavar='encoded tilemap', help='encoded binary tilemap file to decode')
     operation_args.add_argument('--encode', '-e', nargs=1, type=str, metavar='tilemap', help='raw binary tilemap file to encode')
-    arg_parser.add_argument('--wrap', '-w', nargs=1, type=int, metavar='char_count', default=40, help='wrap the stdout output to a number of characters (40 by default; 0 to disable)')
+    arg_parser.add_argument('--wrap', '-w', nargs=1, type=int, metavar='char_count', default=[40], help='wrap the stdout output to a number of characters (40 by default; 0 to disable)')
```

Comment:
Is it wanted that this argument will be parsed as an int, but the default is a list containing one int?

Reply:
Yes, that's an argparse non-intuitive behavior: I learned that options with `nargs=1` still get exposed as a list (containing a single value).

Reply:
Actually your comment prompted me to better read argparse's docs, and there's a better way to specify the args count, which doesn't create a list.

So this is fixed. Thanks!